Use correct aide path variable in the rule descriptions and ocil

ggbecker opened the pull request:
#### Description:

- Use correct aide path variable in the rule descriptions and ocil.

#### Rationale:

- Related to #8305

ggbecker commented:
@dodys please have a look

dodys commented:
we actually use `/usr/sbin/aideinit` for aide_build_database

ggbecker commented:
> we actually use `/usr/sbin/aideinit` for aide_build_database

There might need to have two variables defining these paths then or some jinja if statements for ubuntu to handle this case

jan-cerny commented on linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/rule.yml:
```diff
@@ -7,9 +7,17 @@ title: 'Configure Periodic Execution of AIDE'
+    {{%- if 'ubuntu' not in product %}}
+    <pre>05 4 * * 0 root {{{ aide_bin_path }}} --check</pre>
+    {{%- else %}}
+    <pre>05 4 * * 0 root /usr/bin/aide.wrapper --config /etc/aide/aide.conf --check</pre>
```
Why the variables aide_conf_path and aide_bin_path aren't used here?

ggbecker replied:
aide_conf_path can probably be used... but the aide_bin_path doesn't contain the correct value `aide.wrapper` because other rules already use this variable in a different context... there might be the need to instantiate multiple variables. Briefly checking these aide binaries/wrappers it looks like they differ a lot from distro to distro.

jan-cerny commented:
@ggbecker 

> There might need to have two variables defining these paths then or some jinja if statements for ubuntu to handle this case

So should I expect some changes or are you done?

jan-cerny commented:
@ggbecker bump

jan-cerny commented:
@ggbecker please rebase this and resolve the conflicts

ggbecker commented:
> @ggbecker please rebase this and resolve the conflicts

@jan-cerny I have rebased and reworked some aspects, please check the latest three commits.

@dodys if you can please validate some of the ubuntu changes here. I have switched the aide product variable to `aide_bin_path: "/usr/bin/aide.wrapper"` in all ubuntu products.
